data/connection.go

```diff
@@ -166,6 +192,14 @@ func (c *PostgresSQL) GetOrders(userID int, orderID *int) (model.Orders, error)
 
 			if len(coffee) > 0 {
 				orders[n].Items[i].Coffee = coffee[0]
+
+				ing := []model.CoffeeIngredients{}
+				err := c.db.Select(&ing, "SELECT ingredient_id FROM coffee_ingredients WHERE coffee_id=$1", orders[n].Items[i].Coffee.ID)
+				if err != nil {
+					return nil, err
+				}
+
+				orders[n].Items[i].Coffee.Ingredients = ing
```

Comment:
This will make print the ingredients id like: 
```json
 {
    "id": 4,
    "items": [
      {
        "coffee": {
          "id": 10,
          "name": "my custom vagrante 2",
          "teaser": "Stdin is not a tty",
          "description": "",
          "price": 200,
          "image": "vagrant.png",
          "ingredients": [
            {
              "ingredient_id": 1
            }
          ]
        },
        "quantity": 1
      }
    ]
  }
```

Before was like: 
```json
{
    "id": 63,
    "items": [
      {
        "coffee": {
          "id": 82,
          "name": "my custom vagrante",
          "teaser": "Stdin is not a tty",
          "description": "",
          "price": 200,
          "image": "vagrant.png",
          "ingredients": null
        },
        "quantity": 1
      }
    ]
  }
```